integration-cli/docker_api_inspect_unix_test.go

```diff
@@ -16,7 +16,7 @@ func (s *DockerSuite) TestInspectApiCpusetInConfigPre120(c *check.C) {
 	testRequires(c, cgroupCpuset)
 
 	name := "cpusetinconfig-pre120"
-	dockerCmd(c, "run", "--name", name, "--cpuset", "0-1", "busybox", "true")
+	dockerCmd(c, "run", "--name", name, "--cpuset-cpus", "0-1", "busybox", "true")
```

Comment:
`--cpuset` it's in the deprecated list under `docker run -cpuset` and I removed it here https://github.com/docker/docker/pull/17724/files#diff-ae476143d40e21ac0918630f7365ed3cR91